src/selectors/url-state.js

```diff
@@ -80,6 +81,13 @@ export const getSourceViewFile: Selector<string | null> = (state) =>
   getProfileSpecificState(state).sourceView.sourceFile;
 export const getSourceViewScrollGeneration: Selector<number> = (state) =>
   getProfileSpecificState(state).sourceView.scrollGeneration;
+export const getAssemblyViewIsOpen: Selector<boolean> = (state) =>
+  getProfileSpecificState(state).assemblyView.isOpen;
+export const getAssemblyViewNativeSymbol: Selector<NativeSymbolInfo | null> = (
+  state
+) => getProfileSpecificState(state).assemblyView.nativeSymbol;
+export const getAssemblyViewScrollGeneration: Selector<number> = (state) =>
+  getProfileSpecificState(state).assemblyView.scrollGeneration;
```

Comment:
Completely optional comment, because this is more a style issue than a real problem. So this is more a food for thought:

I wonder if just one selector `getAssemblyView` would be good enough. Then instead of `getAssemblyViewIsOpen(state)` we'd use `getAssemblyView(state).isOpen`. This isn't much worse and would cut down the amount of selectors.

What do you think?

Reply:
Answering to myself: having these individual selectors makes composing `selectors` with `createSelector` easier (as seen in part 3)

Reply:
Oh good point, it didn't come to my mind immediately.